### ISSUE TITLE
discuss spin-off versions of this sim

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/gas-properties#100]
From 5/30/19 design meeting, discuss a "Basics" version of this sim, and maybe a "Diffusion" sim.

High priority, since this needs to be done before dev testing begins.

**Comment by pixelzoom** [comment]
5/30/19 design meeting @arouinfar @ariel-phet @kathy-phet 

I should proceed with "Diffusion" sim, repo name `diffusion`.

**Comment by pixelzoom** [comment]
The diffusion repo is set up and the sim is feature complete.  Not bad for under 2 hours :)  Test drive at [1.0.0-dev.1](https://phet-dev.colorado.edu/html/diffusion/1.0.0-dev.1/phet/diffusion_en_phet.html);

**Comment by kathy-phet** [comment]
Team - Diffusion looks great as a single sim! My only question about Diffusion screen is if a teacher might want a scale on the box, to help somehow have students take data and graph the center of mass marker positions away from center versus time. Not sure what this would look like - small ticks or something. (This would apply to it in the other sim too). Thoughts?

**Comment by pixelzoom** [comment]
Thanks for the suggestion @kathy-phet.  I'm going to move that suggestion to its own issue, since it's a new feature that would apply to both sims.

**Comment by arouinfar** [comment]
Briefly discussed the gas law-focused spinoff with @kathy-phet, and we're thinking to split the sim into two screens.

1. Intro - Ideal screen from `gas-properties`, minus Hold Constant radio options. (Looks like Explore screen, but with magic wall of Ideal screen.)
2. Laws - Identical to Ideal screen in `gas-properties`

As for the sim name, these are the options we're considering
* Gas Laws
* Gas Properties: Basics

I have a slight preference for "Gas Laws" since that is core to the learning goals for this sim. It also is clearly different from Gas Properties.

@kathy-phet pointed out that some teachers may use this sim for introductory exploration without explicitly focusing on the named gas laws or the ideal gas law, so "Gas Laws" may be too narrow of a name.

@ariel-phet any thoughts?

**Comment by ariel-phet** [comment]
I don't really like "Gas Properties: Basics"

I could certainly live with Gas Laws.

I think this is quite akin to our "Waves Intro" sim, so why not "Gases Intro"

**Comment by kathy-phet** [comment]
I like Gases Intro

Kathy

Sent from my iPhone

On Jun 4, 2019, at 12:56 AM, Ariel Paul <notifications@github.com<mailto:notifications@github.com>> wrote:


I don't really like "Gas Properties: Basics"

I could certainly live with Gas Laws.

I think this is quite akin to our "Waves Intro" sim, so why not "Gases Intro"

**Comment by pixelzoom** [comment]
No technical challenges with what's described in https://github.com/phetsims/gas-properties/issues/100#issuecomment-498472587.   There will be a little work to make the "Hold Constant" radio buttons optional, but it's straightforward.  Should be < 3 hours, and I can start as soon as the sim name solidifies (since that's needed to name the repo).

**Comment by arouinfar** [comment]
Thanks for the suggestion @ariel-phet. I also like "Gases Intro"!

@pixelzoom please proceed with `gases-intro`.

**Comment by pixelzoom** [comment]
Sounds great, proceeding.

**Comment by pixelzoom** [comment]
Gases Intro is up and running, see [1.0.0-dev.1]().  We'll need screen icons, but I'll create an issue in the gases-intro repository.   

@arouinfar anything else here? If not, feel free to close.

**Comment by arouinfar** [comment]
@pixelzoom that was quick! Looks like we forgot to specify the `pressureNoise` default as `false` for Gases Intro, so opened https://github.com/phetsims/gases-intro/issues/3.

I don't believe we have any other spin-off sims up our sleeves, so I'll go ahead and close.
